### ISSUE TITLE
How to visual generated images

### ISSUE THREAD

**Issue by MayuOshima** [Godofnothing/pi-GAN#1]
Hello, first thanks for your implementation for pi-GAN!
Could you please tell me how to visualize the generated images? I only see a function called training_epoch_end but because I am new to PyTorch Lightning, I don't know how to use it.

**Comment by Godofnothing** [comment]
Hi, @MayuOshima . Actually `training_epoch_end` produces each `self.sample_every` iterations `self.num_samples` images. They are saved into the `self.output_dir`, which specified in the constructor.

**Comment by MayuOshima** [comment]
Hi, @Godofnothing thanks for your reply! 

Another question, when I do the training using CelebA dataset, the loss_D decreases quickly and the loss_G increase quickly. Have you faced the same problem? Is it because that the batch size(12) is too small? 

Also, it seems that in the paper, gammas and betas used for every SIREN layer are different but in the code, different SIREN layers just share the same gamma and beta if I didn't read the code wrong. So I wonder if there should be some change here.

**Comment by Godofnothing** [comment]
@MayuOshima . Yes at the present the implementation is far from complete - since two key features are not implemented - `FiLM` conditioning needs to be separate for each MLP layer, and the generator doesn't have the part, which takes the embedding of the viewing direction. Concerning the discriminator loss - yes, we have faced such a problem, we tried to put small learning rate for discriminator. Actually I have abandonded this project so far, since I've contacted one of the authors of `\pi-GAN` paper and they are going to release their code in public access. So far, as I've understood there is cleaning and tidying of the code stage. Maybe, If their release would be postponed to far future and I'll have time, I'll return back to this project and improve it.
